os/board/rtl8730e/src/component/os_dep/osdep_service_memory.c

```diff
@@ -6,15 +6,21 @@
 #include <osdep_service.h>
 #include <stdio.h>
 
+/* For Smart, need to align malloc to 64bytes (cache line size of AP) for cache operations */
+#if CONFIG_AMEBAD2
```

Comment:
Could you let me know where this config is defined?

Reply:
Hi, this config is defined in RTK's [platform_autoconf.h](https://github.com/Samsung/TizenRT/blob/master/os/board/rtl8730e/src/project/realtek_amebaD2_va0_example/inc/inc_ap/platform_autoconf.h)

Reply:
1. The title says "RTL8730E". Is that `AMEBAD2`? not `AMEBASMART`?
2. You define a proper config and don't define other chip. In other word,
````
#define CONFIG_AMEBAD2 1
// no define for CONFIG_AMEBALITE
```
Then, you should use `#ifdef` and `#elif defined` instead of `#if` and `#elif`.
If you use like below
```
#define CONFIG_AMEBAD2 1
#define CONFIG_AMEBALITE 0
```
Then, current is correct.

Reply:
Hi @sunghan-chang,  since the file is in the rtl8730e path, I will remove the condition for checking the chip as it would be unnecessary to check.